plugins/put-a-file-with-ftp/impl/src/test/java/org/pentaho/di/job/entries/ftpput/JobEntryFTPPUTTest.java

```diff
@@ -89,6 +114,43 @@ public void createFtpClient_UsesTimeoutIfSet() throws Exception {
     verify( ftpClient ).setTimeout( 10 );
   }
 
+  @Test
+  public void testLocalDirDoesntExist() throws Exception{
```

Comment:
This test case `testLocalDirDoesntExist()` doesn't make any sense.

 The new code you added throws an exception
`throw new FTPException(...`
But yet here you aren't checking for the thrown FTOException anywhere.

And somehow, the `execute()` return with a result.

Reply:
obsolete case in source code was removed with https://github.com/pentaho/pentaho-kettle/commit/42a31cf00e0008515d6a51939fcac7cd624e1075